src/formats.ts

```diff
@@ -48,6 +48,8 @@ enum SupportedLocale {
   Dutch = 'nl',
   Portugese = 'pt-BR',
   Russian = 'ru',
+  ChineseTraditional = 'zh-CN',
```

Comment:
You checked that these are the locale keys we're getting from GraphQL in web, right?